browser/brave_wayback_machine/brave_wayback_machine_util.cc

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_wayback_machine/brave_wayback_machine_util.h"
+
+#include "base/containers/flat_set.h"
+#include "brave/browser/brave_wayback_machine/brave_wayback_machine_infobar_delegate.h"
+#include "brave/common/pref_names.h"
+#include "chrome/browser/profiles/profile.h"
+#include "components/prefs/pref_service.h"
+#include "content/public/browser/web_contents.h"
+
+void CheckWaybackMachineIfNeeded(content::WebContents* contents,
```

Comment:
I'm not really following what this method does, can you add comments to the header file?